### PR TITLE
Add etcd healthcheck port to requirements

### PR DESCRIPTION
**Pull request by manuelbuil**
Issue: https://github.com/rancher/rke2/issues/4602

### CODE REVIEW

File: docs/install/requirements.md [hunk 1]
```diff
@@ -95,6 +95,7 @@ If you wish to utilize the metrics server, you will need to open port 10250 on e
 | TCP | 10250 | RKE2 server and agent nodes | kubelet
 | TCP | 2379 | RKE2 server nodes | etcd client port
 | TCP | 2380 | RKE2 server nodes | etcd peer port
+| TCP | 2381 | RKE2 server nodes | etcd healthcheck port
```
[the remark this answers is not in the document]
Comment (manuelbuil) [reply docs/install/requirements.md:98]: Is the metrics port the same as the healthcheck: https://github.com/rancher/rke2/blob/master/pkg/podexecutor/staticpod.go#L533?

**Comment by dereknola** [comment]
@manuelbuil this still relevant?

File: docs/install/requirements.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (brandond) [reply docs/install/requirements.md:98]: It's the metrics port, which we use for health checking

https://github.com/k3s-io/k3s/blob/master/pkg/etcd/etcd.go#L828-L835

**Comment by manuelbuil** [comment]
> @manuelbuil this still relevant?

Yes, I think so. Do you have concerns?

**Comment by dereknola** [comment]
No, you just never addressed Brad's comment. I was going thru all the open PRs.

**Comment by manuelbuil** [comment]
> No, you just never addressed Brad's comment. I was going thru all the open PRs.

Ah, right! Sorry about that, PTO +crazyness + waiting on Brad's reply

**Review by rbrtbnfgl — APPROVED** [review]

**Review by dereknola — APPROVED** [review]

**Pull request merged**